csharp/src/Ice/Communicator-Properties.cs

```diff
@@ -144,6 +144,64 @@ internal PropertyValue(string v, bool u)
             }
         }
 
+        /// <summary>
+        /// Gets the value of a property as a TimeSpan. If the property is not set, returns null.
+        /// The value must be an integer followed immediately by a time unit of 'ms', 's', 'm', or 'h'.
+        /// These correspond to milliseconds, seconds, minutes, and hours, respectively.
+        /// e.g. 50ms, 3m
+        /// </summary>
```

Comment:
I think our convention is to write comment tags inline, since that's what C# standardly does apparently.